demisto_sdk/commands/common/errors.py

```diff
@@ -1363,6 +1364,12 @@ def readme_equal_description_error():
         return 'README.md content is equal to pack description. ' \
                'Please remove the duplicate description from README.md file.'
 
+    @staticmethod
+    @error_code_decorator
+    def metadata_url_invalid():
+        return 'The metadata url leads to a GitHub repo instead of a support page. ' \
```

Comment:
```suggestion
        return 'The metadata URL leads to a GitHub repo instead of a support page. ' \
```

Reply:
Let's also send them to the right doc for help https://xsoar.pan.dev/docs/packs/packs-format#pack_metadatajson